### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by Lykhoyda**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,9 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           cache: 'yarn' # cache node modules
-          node-version: lts
+          node-version: latest
```
Comment (BeroBurny) [inline .github/workflows/ci.yml]: it should stay on `lts`

**Review by BeroBurny — CHANGES_REQUESTED** [review]
the node should be updated to version `18`
as current `lts` is 18
https://nodejs.dev/en/about/releases/

File: .github/workflows/ci.yml (hunk 1 above)
Comment (BeroBurny) [reply .github/workflows/ci.yml]: `latest` is 19? and `lts` 18?

Comment (Lykhoyda) [reply .github/workflows/ci.yml]: just "lts" doesn't work, the examples are there: https://github.com/actions/setup-node#supported-version-syntax
I will try "lts/*"

Comment (BeroBurny) [reply .github/workflows/ci.yml]: then better to use the same version as in `.nvmrc`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -15,7 +15,9 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           cache: 'yarn' # cache node modules
-          node-version: lts
+          node-version: lts/*
```
Comment (BeroBurny) [inline .github/workflows/ci.yml]: this can be the same as in `.nvmrc`

**Review by BeroBurny — COMMENTED** [review]
One more important thing `.nvmrc` need to be updated

**Review by BeroBurny — APPROVED** [review]
:package:

**Pull request merged**